api/v1/authenticator_types.go

```diff
@@ -39,6 +39,9 @@ type AuthenticatorSpec struct {
 	// Authentication configures back-end authentication for this authenticator
 	Authentication Auth `json:"authentication"`
 
+	// ServiceAccount sa to used by authenticator daemonset to update status
+	ServiceAccount string `json:"serviceAccount"`
```

Comment:
Does this need to be configurable by the end-user and put in every CR?  Can this just be handled internally by the controller?